src/test/spec/retryable_writes/mod.rs

```diff
@@ -63,7 +64,15 @@ async fn run_spec_tests() {
                     .expect(&test_case.description);
             }
 
-            if let Some(ref fail_point) = test_case.fail_point {
+            if let Some(ref mut fail_point) = test_case.fail_point {
+                // TODO: DRIVERS-1385 remove this logic for moving errorLabels to the top level.
```

Comment:
I ended up just changing the failpoints to be accurate. This had the benefit of not requiring any driver changes, and it's likely how the tests will be fixed in the future.